rc/lsp.kak

```diff
@@ -266,6 +268,14 @@
     lsp-menu %arg{@}
 }
 
+define-command -hidden lsp-follow-document-link -params 1.. -docstring "Called on :lsp-document-link" %{
+    lsp-menu %arg{@}
```

Comment:
Adding override points that are unlikely to ever be used feels somewhat odd. (I can imagine someone might override lsp-menu but not so much the individual lsp-menu users).

Probably it's fine to keep it, because it's consistent with the existing structure.
Not sure yet.